controller/src/main/java/com/softserve/borda/controllers/UserController.java

```diff
@@ -30,7 +30,7 @@
 
     private final JwtConvertor jwtConvertor;
 
-    @GetMapping
+    @GetMapping(value = "getAll")
```

Comment:
Bad resource naming.
Please read https://restfulapi.net/resource-naming/ doc and change naming.

Reply:
Fixed in FixResourcesPaths branch/ticket, already merged in main